.github/workflows/pr-build-live-branch.yml

```diff
@@ -0,0 +1,85 @@
+name: Build Live Branch
+on:
+  push:
+  pull_request:
+    types: [opened,edited,labeled] 
```

Comment:
I don't think these types are required

> if no activity types are specified, the workflow runs when a pull request is opened or reopened or when the head branch of the pull request is updated

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
Thanks... I had these in place while I was testing for easier retriggering, but forgot to remove them. Updated!

Reply:
Did you push? I'm not seeing this line updated

Reply:
Doh, I had overwrote the change I made on GH's UI with a force-push to fix a failing test. Re-fixed now 😄